### DISCUSSION TITLE
Webhook support

### DISCUSSION THREAD

**Discussion by anoop-b** [LinkoraApp/Linkora#38] [Ideas]
Some sort of sync support would be nice. Webhook support would be a good start imo.

**Comment by sakethpathike** [comment]
I'm working on a server that can be self-hosted, other clients (if any) can read the changes made by other clients (a typical web socket implementation), if the client isn't online, it can later read it via a certain endpoint (didn't implemented this yet, will implement a typical _delta sync_ stuff)

**Reply by aadnan181** [reply]
I was able to setup cross sync between my android device and PC by self-hosting. But I wanted to know is there any better way to add bookmarks in PC rather than manually copying the link and adding it to the linkora app? Any browser extensions of some sort?

**Reply by sakethpathike** [reply]
@aadnan181 

> ... is there any better way to add bookmarks in PC rather than manually copying the link and adding it to the linkora app? Any browser extensions of some sort?

no, not yet.

**Reply by aadnan181** [reply]
Any plans for future release?

**Reply by sakethpathike** [reply]
@aadnan181 

> Any plans for future release?

yes but not any time soon (I'll see about it once i release the next version)
